### PR TITLE
tgl: ipc4: zephyr: move to native drivers

### PR DESCRIPTION
**Pull request by juimonen**
Set tgl ipc4 overlays to use zephyr native drivers.

Signed-off-by: Jaska Uimonen <jaska.uimonen@intel.com>

### CODE REVIEW

File: app/overlays/tgl-h/ipc4_overlay.conf [hunk 1]
```diff
@@ -20,3 +20,30 @@ CONFIG_ZEPHYR_LOG=y
...
+
+CONFIG_INTEL_ADSP_IPC=y
+
+CONFIG_DMA=y
+CONFIG_DMA_DW_LLI_POOL_SIZE=100
+CONFIG_DMA_DW_HW_LLI=y
+CONFIG_DMA_DW_FIFO_PARTITION=y
+CONFIG_DMA_INTEL_ADSP_GPDMA_HAS_LLP=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```
Comment (lgirdwood) [inline app/overlays/tgl-h/ipc4_overlay.conf:49]: newline ?

Comment (juimonen) [reply app/overlays/tgl-h/ipc4_overlay.conf:49]: added

**Review by aborisovich — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Comment by aborisovich** [comment]
CI prepared in #6697

**Review by abonislawski — APPROVED** [review]
Still one new line needed

**Pull request merged**